libcxx/docs/UsingLibcxx.rst

```diff
@@ -147,6 +147,35 @@ IWYU, you should run the tool like so:
 If you would prefer to not use that flag, then you can replace ``/path/to/include-what-you-use/share/libcxx.imp``
 file with the libc++-provided ``libcxx.imp`` file.
 
+.. _assertions-mode:
+
+Enabling the "safe libc++" mode
```

Comment:
I wonder whether we should bring back this documentation. It's one thing for the safe mode to still work, however I feel like leaving this documentation here might just be confusing to users.

Reply:
Removed. Should we also remove documentation for `LIBCXX_ENABLE_ASSERTIONS` in `BuildingLibcxx.rst`?